CRM/Contact/BAO/Query.php

```diff
@@ -4971,7 +4972,9 @@ public function getCachedContacts($cids, $includeContactIds) {
     $limit = '';
     $query = "$select $from $where $groupBy $order $limit";
 
-    return CRM_Core_DAO::executeQuery($query);
+    $result = CRM_Core_DAO::executeQuery($query);
+    CRM_Core_DAO::disableFullGroupByMode();
```

Comment:
this doesn't look right @eileenmcnaughton

Reply:
erm yep, fixed